sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -3,27 +3,89 @@
 
 #pragma once
 
+#include <string>
+#include <vector>
+
 #include <internal/contract.hpp>
```

Comment:
>hpp [](start = 28, length = 3)

Do we have guideline of whether we should use `.h` or `.hpp`?

Reply:
@gilbertw

Reply:
Do we have a guideline of when to use `#include <>` and when to use `#include ""`